decentraland/unity-explorer
# Feat/single emoji chat bubble

## What does this PR change?

Fix #741 
Makes chat bubble bigger above players when sending a single emoji, the issue says on chat panel as well but this pr fixes only the nametag chat bubble

Fix #807 
Adds the enable/disable of nametags with the key "N"

...

## How to test the changes?


Test #741 
1. Launch the explorer
2. Chat with a single emoji
3. Check the chat bubble is bigger for a single emoji

Test #807 
1. Press N
2. Check if the nametags disappear
3. Press it again
4. Check if the nametags re-appear

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
